tests/py3-ignores.txt

```diff
@@ -27,8 +25,6 @@ tests/test_engine.py
 tests/test_http_cookies.py
 tests/test_logformatter.py
 tests/test_mail.py
-tests/test_pipeline_files.py
-tests/test_pipeline_images.py
```

Comment:
Pipelines tests don't look complete, and I think we shouldn't uncomment them even if they are passing; I'm not convinced these modules are ported. E.g. headers are now bytes, but pipeline tries to check for Referrer header as a string.